core/trino-main/src/main/java/io/trino/connector/ConnectorManager.java

```diff
@@ -165,12 +165,14 @@ public synchronized void stop()
 
         for (Map.Entry<CatalogName, MaterializedConnector> entry : connectors.entrySet()) {
             Connector connector = entry.getValue().getConnector();
-            try (ThreadContextClassLoader ignored = new ThreadContextClassLoader(connector.getClass().getClassLoader())) {
-                connector.shutdown();
-            }
-            catch (Throwable t) {
```

Comment:
Previously this was also catching exceptions from `ThreadContextClassLoader.close()`

Reply:
moved inner try-block into outer one